### PR TITLE
Remove instructions: "don't write a for loop..."

### PR DESCRIPTION
**Pull request by eschwartz**
I had a few students thinking that they shouldn't start out by writing a for loop,
and got stuck :-o

### CODE REVIEW

File: client.js [hunk 1]
```diff
@@ -33,8 +33,7 @@ const employees = [
 
 // YOU SHOULD NOT NEED TO CHANGE ANYTHING ABOVE THIS POINT
 
-// Take small steps! Don't write a for loop and two functions that do all of the calculations right away.
-// This problem is massive! Break the problem down. Use the debugger.
```
Comment (eschwartz) [inline client.js:37]: we haven't taught the debugger yet

**Comment by kdszafranski** [comment]
I think we could go further here if we wanted. What I often see (saw - in person), is that some people would get fixated on the loop and forget that in order to process 6 employees, they do need working logic to do even 1. So, get 1 working well and then worry about the array.

**Comment by eschwartz** [comment]
@kdszafranski I'm definitely seeing that too. Just that process of calling a function inside that loop had maybe 20% of my class stuck for way longer than I'd like.

Are you thinking a bit more hand-holding here, around process? Like, make suggestions on how to iterate? ("Start by passing a single employee to the function... setup a return object... etc.")

**Comment by kdszafranski** [comment]
@eschwartz Yes, but I'm not sure in the instructions/code is the best place tho. I think it's a teaching moment.

Maybe we add a step in the instructions that is about Planning, and make those suggestions there

**Comment by eschwartz** [comment]
Let's maybe talk about this in person. Added a _meeting_ label

**Comment by DoctorHowser** [comment]
To discuss with @kdszafranski

**Review by kdszafranski — APPROVED** [review]
approved by committee

**Pull request merged**